frontend/src/bundles/studio/components/player-controllers/player-controllers.tsx

```diff
@@ -0,0 +1,100 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+import {
+    Flex,
+    Icon,
+    IconButton,
+    Tooltip,
+} from '~/bundles/common/components/components.js';
+import { useCallback, useState } from '~/bundles/common/hooks/hooks.js';
+import { IconName } from '~/bundles/common/icons/icons.js';
+
+import { TimeDisplay } from './components/components.js';
+
+const PlayerControllers: React.FC = () => {
+    // Mocked data. Update later
+    const currentTime = 5;
+    const duration = 10;
+
+    const [isPlaying, setIsPlaying] = useState<boolean>(false);
+
```

Comment:
It's probably better to pass isPlaying as a component property, since the parent component will handle that logic. The same applies to currentTime and duration.

Reply:
This task about adding view, we will develop logic separately